.github/workflows/metabase-deploy.yaml

```diff
@@ -4,6 +4,11 @@ on:
   workflow_call:
     secrets:
       OPENSHIFT_SERVER: { required: true }
+      OPENSHIFT_TOKEN: { required: true }
+      OPENSHIFT_METABASE_NAMESPACE: { required: true }
```

Comment:
nit: can use `secrets: inherit` here as well

Reply:
Ah cool, I am going to try that before I merge this in thanks